docker-compose.prod.yml

```diff
@@ -0,0 +1,9 @@
+version: '2'
+
+services:
+  firefly-app:
+    container_name: firefly-iii
+    environment:
+      - FF_APP_ENV=production
+    volumes:
+      - ./storage:/var/www/firefly-iii/storage
```

Comment:
Are you sure this is needed? The folder inside `/var/www/firefly-iii/` already has a full storage in it (since the dockerfile copies the complete project inside the container). I tested it with a named volume directly and seems to work fine. The only caveat is that is harder to check the app logs.